content/en/bits_ai/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Bits AI
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+    - link: "https://www.datadoghq.com/product/platform/bits-ai/"
+      tag: "Product page"
+      text: "Bits AI"
+    - link: "https://www.datadoghq.com/blog/datadog-bits-generative-ai/"
+      tag: "Blog post"
+      text: "Introducing Bits AI, Your New DevOps Copilot"
+    - link: "https://www.datadoghq.com/about/latest-news/press-releases/datadog-announces-bits-an-ai-assistant-to-help-engineers-quickly-resolve-application-issues/"
+      tag: "Press release"
+      text: "Bits AI announcement"
+---
+
+<div class="alert alert-info">Bits AI is in private beta.</div>
+
+Bits AI is a generative AI interface that helps you identify, remediate, and summarize issues in your applications and infrastructure. It surfaces faulty deployments, Watchdog anomalies, incidents, alerts, and more.
+
+## Features
+
+### Query data in natural language
+
+Bits AI supports natural language querying for logs, APM traces, infrastructure data, and cloud cost.
+
+In Datadog or in Slack, you can ask Bits AI questions such as
+- What is going on with example-service?
+- Are there any issues with example-service's dependencies?
+- Who is on call for example-service?
+- Find me the example-service dashboard.
+
+Bits AI also expands on issues with upstream and downstream dependencies. This feature works best if your APM services are tagged by **team** and **service**.
+
+{{< whatsnext desc="Additional documentation:">}}
```

Comment:
Is it possible to include these links in line vs having them in a separate section like this? The getting started page is very important and I worry that it makes it difficult for our customers to find that info. I would prefer having the contents of that page in a prerequisites section on this main page.

Reply:
See my comments elsewhere for why the Getting Started content isn't inline above this -- but I added a link at the top of the page. I used a section like this because its nav style is sometimes easier for the reader to see than inline links, so I do recommend keeping it as well, but using both doesn't hurt!